Form/Handler/AuthorizeFormHandler.php

```diff
@@ -71,26 +71,34 @@ public function isRejected()
         return !$this->form->getData()->accepted;
     }
 
+    /**
+     * @return bool|void
+     */
     public function process()
     {
         $request = $this->getCurrentRequest();
-        if (null !== $request) {
-            $this->form->setData(new Authorize(
-                $request->request->has('accepted'),
-                $request->query->all()
-            ));
-
-            if ('POST' === $request->getMethod()) {
-                $this->form->handleRequest($request);
-                if ($this->form->isValid()) {
-                    $this->onSuccess();
-
-                    return true;
-                }
-            }
+
+        if (null === $request) {
+            return false;
+        }
+
+        $this->form->setData(new Authorize(
+            $request->request->has('accepted'),
+            $request->query->all()
+        ));
+
+        if ('POST' !== $request->getMethod()) {
+            return;
```

Comment:
Why not false here?

Reply:
I will be fixing this one. Dunno how I missed it.